### PR TITLE
Add beginner guide

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by leeyeh** [comment]
Deploy preview for *lc-docs-en* ready!

Built with commit 8a0fc9c772462d3723aeab6eda39e99edf8f8687

https://deploy-preview-52--lc-docs-en.netlify.com

File: views/realtime-guide-beginner.md [hunk 1]
```diff
@@ -5,3 +5,3048 @@
...
+- [In the third documentation](realtime-guide-senior.html), we will introduce the security features offered by our services, including third-party signing mechanism, permission management of members, and blacklisting. We will also go over the usage of chat rooms and temporary conversations. By the end of this documentation, you will get a set of skills to improve the security and usability of your app, as well as to build conversations that serve different purposes.
+- [In the last documentation](realtime-guide-systemconv.html), we will introduce the usage of hooks and system conversations, plus how you can build your own chatbots based on them. By the end of this documentation, you will learn how you can have your app spontaneously chat with users.
```
Comment (weakish) [inline views/realtime-guide-beginner.md]: 「you will learn how you can have your app spontaneously chat with users.」这里中文是「希望可以满足业务层多种多样的扩展需求」

File: views/realtime-guide-beginner.md [hunk 2]
```diff
@@ -5,3 +5,3048 @@
...
+
+We aim these documentations to not only help you complete the functions you are currently building but also give you a better understanding of all the things LeanMessage can do (which you will find helpful when you plan to add more features into your app).
+
+> Before you continue:
+>
+> Take a look at [LeanMessage Overview](realtime_v2.html) if you haven’t done yet. Also make sure you have already followed [SDK Installation](start.html) to install and initialize the SDK for the platform (language) you are using.
```
Comment (weakish) [inline views/realtime-guide-beginner.md]: haven't done yet -> haven't done it yet

File: views/realtime-guide-beginner.md [hunk 3]
```diff
@@ -5,3 +5,3048 @@
...
+    IDictionary<string, object> options = null);
+```
+
+Although SDKs for different languages/platforms share different interfaces, they take in the similar set of parameters when creating a conversation:
+
+
+1. `members`: Required; includes the initial list of members in the conversation. The initiator of the conversation is included by default, so `members` does not have to include the `clientId` of the current user.
+2. `name`: The name of the conversation; optional. The code above puts “Tom & Jerry” for it.
+3. `attributes`: The custom attributes of the conversation; optional. The code above does not specify any attributes. If you ever specify them for your conversations, you can retrieve them later with `AVIMConversation`. Such attributes will be stored in the `attr` field of the `_Conversation` table.
+4. `unique`/`isUnique` or `AVIMConversationOptionUnique`: Markes if the conversation is unique; optional.
```
Comment (weakish) [inline views/realtime-guide-beginner.md]: Markes -> Marks

File: views/realtime-guide-beginner.md [hunk 4]
```diff
@@ -5,3 +5,3048 @@
...
+/**
+ * Deregister handler for specific message type.
+ *
+ * @param clazz
+ * @param handler
+ */
+public static void unregisterMessageHandler(Class<? extends AVIMMessage> clazz, MessageHandler<?> handler);
+```
+
+Different handlers can be registered or deregistered for different message types (including those defined by yourself). These handles should be set up when initializing the app.
```
Comment (weakish) [inline views/realtime-guide-beginner.md]: These handles -> These handlers

File: views/realtime-guide-beginner.md [hunk 5]
```diff
@@ -5,3 +5,3048 @@
...
+So when handlers are specified for `AVIMTypedMessage` (and its subtypes) and a global `defaultHandler` is also specified, if the sender sends a general `AVIMMessage` message, the receiver will have its handler in `AVIMMessageManager#registerDefaultMessageHandler()` triggered; if the sender sends a message of `AVIMTypedMessage` (or its subtype), the receiver will have its handler in `AVIMMessageManager#registerMessageHandler()` triggered.
+
+{{ docs.langSpecEnd('java') }}
+
+{{ docs.langSpecStart('cs') }}
+
+The C# SDK handles new messages with `OnMessageReceived`. There are **two levels** here:
+
+* The first level is applied to `AVIMClient` which serves to get you notified that there are messages coming in. This is especially helpful when there is no local `AVIMConversation` (like when a user just logged in) and a new message is sent to a conversation involving the current user. `AVIMClient.OnMessageReceived` would take in the messages, but has nothing to do with the type of the message.
+*
```
Comment (weakish) [inline views/realtime-guide-beginner.md]: no array value -> on array value

Comment (weakish) [inline views/realtime-guide-beginner.md]: `AVIMClient.open(callback)` 后面需要换行

**Review by weakish — CHANGES_REQUESTED** [review]
有一些中文文档的 typo 提在 leancloud/docs#2946，需要同步下。

File: views/realtime-guide-beginner.md (hunk 1 above)
Comment (ghost) [reply views/realtime-guide-beginner.md]: 感觉中文的这句话好抽象啊，想了半天不知道怎么用英文讲……😂

Comment (weakish) [reply views/realtime-guide-beginner.md]: you will learn how you can make your app extensible and adapted to a wide variety of  requirements

**Review by weakish — DISMISSED** [review]

File: views/realtime-guide-beginner.md (hunk 1 above)
Comment (ghost) [reply views/realtime-guide-beginner.md]: 👍👍👍

**Review by weakish — APPROVED** [review]

**Pull request merged**